src/patternfly/components/ContextSelector/context-selector.scss

```diff
@@ -118,6 +128,38 @@
     }
   }
 
+  &.pf-m-plain {
+    --pf-c-context-selector__toggle--PaddingRight: var(--pf-c-context-selector__toggle--m-plain--PaddingRight);
+    --pf-c-context-selector__toggle--PaddingLeft: var(--pf-c-context-selector__toggle--m-plain--PaddingLeft);
+
+    &:not(.pf-m-text) {
+      justify-content: center;
```

Comment:
Let's add `display: inline-block` here for the `.pf-m-plain` (non-text) variant, then we can remove `justify-content` since that property doesn't apply to inline-block elements.